### PR TITLE
Shape interface for Circle and Perimeter structs

### PR DESCRIPTION
**Pull request by AlenaTMW**
Please, check my homework.

### CODE REVIEW

File: lesson_04/main.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+type Shape interface {
+	Area() float64
```
Comment (rat6666) [inline lesson_04/main.go]: no need to use exported names if you only use them in this package

File: lesson_04/main.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	width  float64
+}
+
+func DescribeShape(s Shape) {
+	fmt.Println(s)
+	fmt.Printf("Area: %.2f\n", s.Area())
+	fmt.Printf("Perimeter: %.2f\n", s.Perimeter())
+}
+
+//func (c Cat) String() string {return fmt.Sprintf("Cat has %d legs\n", c.legs)}
```
Comment (rat6666) [inline lesson_04/main.go]: pleace remove unnecessary comment

File: lesson_04/main.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+func (c Circle) String() string {
+	return fmt.Sprintf("\nCirlce: radius %.2f", c.radius)
+}
+
+func (r Rectangle) String() string {
+	return fmt.Sprintf("\nRectangle with height %.2f and width %.2f", r.height, r.width)
+}
+
+func main() {
+	// choose your own dimensions
```
Comment (rat6666) [inline lesson_04/main.go]: pleace remove unnecessary comment

File: lesson_04/main.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+type Shape interface {
+	Area() float64
+	Perimeter() float64
```
Comment (rat6666) [inline lesson_04/main.go]: the same here

**Review by rat6666 — CHANGES_REQUESTED** [review]

**Review by rat6666 — APPROVED** [review]

**Review by dimonl — APPROVED** [review]

**Pull request merged**